jobs/index.js

```diff
@@ -1,25 +1,36 @@
 // @ts-check
 
-const VQueue = require('../components/bull-queue');
 const logger = require('../components/logger');
-const { QUEUE } = require('../constants/queue');
 
-// import handlers
+// rabbit
+const { getListQueueRabbit } = require('./rabbit-queue');
+const { getListQueueBull } = require('./bull-queue');
+const { QUEUE } = require('../constants/queue');
 const { pingHandler } = require('./consumers/ping');
+const { handlerNotification } = require('./consumers/noitification');
+const { TYPE_QUEUES } = require('../constants/common');
 
 const settings = {
+    // bull
     [QUEUE.Ping]: {
         concurrency: 1,
         handler: pingHandler,
+        type: TYPE_QUEUES.BULL,
+    },
+
+    // rabbit
+    [QUEUE.Notification]: {
+        handler: handlerNotification,
+        type: TYPE_QUEUES.RABBIT,
     },
 };
 
-const listQueue = Object.keys(settings).reduce((init, queueName) => {
-    const { concurrency } = settings[queueName];
-    return Object.assign(init, {
-        [queueName]: new VQueue(queueName, concurrency),
-    });
-}, {});
+async function getListQueue() {
+    const listQueueRabbit = await getListQueueRabbit(settings);
```

Comment:
show loop through settings inside this function and check flag `type` to create queue